web/backend/v2/src/test/java/com/tourneygen/web/Controllers/ErrorControllerTests.java

```diff
@@ -0,0 +1,50 @@
+package com.tourneygen.web.Controllers;
```

Comment:
Can we move the tests to the test directory? If we can't because of how springboot works, can we create a tests directory as high as possible?

Reply:
It seems i cant move this directory